src/rvpt/rvpt.cpp

```diff
@@ -182,10 +183,24 @@ void RVPT::update_imgui()
         ImGui::PushItemWidth(80);
         ImGui::SliderInt("AA", &render_settings.aa, 1, 64);
         ImGui::SliderInt("Max Bounce", &render_settings.max_bounces, 1, 64);
```

Comment:
Could make it `Max Bounces` and make AA to SPP or Samples, AA isn't exactly what it does even if it achieves the effect.